src/static/js/modules/jquery/cf_formValidator.js

```diff
@@ -0,0 +1,185 @@
+/* ==========================================================================
+   jquery.cf_formValidator
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+var validate = require( 'validate' );
+
+function init() {
+  var validator = {
+    defaults: {
+      inputs: [
+        'input[type!="hidden"]:enabled',
+        'textarea:enabled',
+        'select:enabled'
+      ],
+      types: [
+        'required',
+        'email',
+        'checkgroup',
+        'radiogroup'
+      ],
+      onFailure: function( event, fields ) {},
+      onSuccess: function( event, fields ) {}
+    },
+
+    // Validate an individual input for each of the set types
+    // @param  {object} $input The jQuery object of the input
+    // @param  {string} value  The evaluated value of the input
+    // @return {object}        The status of each of the tested types
+    _validateInput: function( $input, value ) {
+      var status = {};
+
+      $.each( validator.settings.types, function( i ) {
+        var validation;
+        var type        = this;
+        var notRequired = ( type === 'required' && !$input.prop( 'required' ) );
+        var notEmail    = ( type === 'email' && $input.attr( 'type') !== 'email' );
+        var notRadio    = ( type === 'radiogroup' && $input.attr( 'type' ) !== 'radio' );
+        var notCheckbox = ( type === 'checkgroup' && $input.attr( 'type' ) !== 'checkbox' );
+
+        if ( notRequired || notEmail || notRadio || notCheckbox ) {
+          status[type] = null;
+          return;
```

Comment:
My comment was in error, since this is a return on a nested function the return type can be different than L:56. Just set it to `return false;` per docs on https://api.jquery.com/jquery.each/